pkg/slo-controller/noderesource/plugins/util/util.go

```diff
@@ -270,8 +275,17 @@ func GetNodeCapacity(node *corev1.Node) corev1.ResourceList {
 
 // GetNodeSafetyMargin gets node-level safe-guarding reservation with the node's allocatable
 func GetNodeSafetyMargin(strategy *configuration.ColocationStrategy, nodeCapacity corev1.ResourceList) corev1.ResourceList {
-	cpuReserveQuant := util.MultiplyMilliQuant(nodeCapacity[corev1.ResourceCPU], getReserveRatio(*strategy.CPUReclaimThresholdPercent))
-	memReserveQuant := util.MultiplyQuant(nodeCapacity[corev1.ResourceMemory], getReserveRatio(*strategy.MemoryReclaimThresholdPercent))
+	defaultstrategy := sloconfig.DefaultColocationStrategy()
```

Comment:
is it necessary to merge the default here?

Reply:
@saintube The reason for this change is that when I run the test unit `Test_getNodeReservation`, if `CPUReclaimThresholdPercent` or `MemoryReclaimThresholdPercent` not configured, the code will panic since `CPUReclaimThresholdPercent` is nil. I will add the relevant test unit later.

Reply:
Perhaps the default config should be prepared in the UT instead of the main function.

Reply:
fixed.